sky/utils/schemas.py

```diff
@@ -50,6 +50,11 @@ def get_resources_schema():
                     'additionalProperties': {
                         'type': 'number'
                     }
+                }, {
+                    'type': 'array',
+                    'items': {
+                        'type': 'string',
+                    }
```

Comment:
Will there be a difference between `[V100:1, A100:1]` vs `{V100:1, A100:1}` here?

Reply:
@Michaelvll  'array` only applies to `['A100-40GB:1', 'V100:1', 'K80:1', 'T4:1']`
`{'A100-40GB:1', 'K80:1', 'V100:1', 'T4:1', 'T4:4'}` will be read as a dictionary.

Reply:
I see, in that case, which of the schemas here represents the `{V100:1, A100:1}`?

Reply:
`{V100:1, A100:`}` is not standard json, so it got checked with `type: string`

Reply:
Oh, that is a bit suprising as our code will read this as a dict as you mentioned. It worth a comment here.

Reply:
@Michaelvll let me add a comment there. thanks!